source/includes/fact-oplog-background.rst

```diff
@@ -4,14 +4,19 @@ to the data on the destination cluster.  When operations
 that ``mongosync`` has not applied roll off the ``oplog`` 
 on the source cluster, the sync fails and ``mongosync`` exits.
 
-During the initial sync, ``mongosync`` may apply operations at a slower
-rate due to the load imposed by copying documents concurrently.
-After ``mongosync`` completes the initial sync, it applies changes 
-faster and is more likely to maintain a position in the ``oplog``
-that is close to the real-time writes occuring on the source cluster.
+Starting in version 1.5.0, {+c2c-product-name+} enables Oplog Rollover
+Resilience (ORR) which allows ``mongosync`` to apply changes on the
+source cluster to the destination cluster during the initial sync. ORR
+increases the resilience of ``mongosync`` to oplog rollover but does not
+prevent rollover entirely. For more information on ORR, see the :ref:`Oplog
+Rollover Resilience release note <1.5.0-c2c-release-notes>`.
+
+If you anticipate that you will sync from a source cluster with a high
```

Comment:
Same comments as above.
Although could you use the include as the answer to the FAQ?

Reply:
Because the includes seems to provide more information that needed to answer the faq, I put a shorter version of the includes in the faq section starting with the cases in which a user might need to increase the size of the oplog. My thought is that this more directly answers the question. What do you think?

Reply:
I also added a link to [Workloads that Might Require a Larger Oplog Size](https://www.mongodb.com/docs/manual/core/replica-set-oplog/#workloads-that-might-require-a-larger-oplog-size) per Ajay's request on JIRA and will need to open a ticket to add a ref to the Replica Set Oplog page